tests/cupy_tests/random_tests/test_generator.py

```diff
@@ -1188,9 +1188,21 @@ def test_randint_1(self):
     def test_randint_2(self):
         self.generate(3, 4, size=(3, 2))
 
-    def test_randint_3(self):
+    def test_randint_empty1(self):
         self.generate(3, 10, size=0)
 
+    def test_randint_empty2(self):
+        self.generate(3, size=(4, 0, 5))
+
+    def test_randint_overflow(self):
+        self.generate(numpy.int8(-100), numpy.int8(100))
+
+    def test_randint_float1(self):
+        self.generate(-1.2, 3.4, 5)
```

Comment:
How about adding cases like these:
* (-1.8, 3.9)  # `round` would result in different range
* (-3.8, -1.2)  # negative number at the upper bound

Reply:
Thanks for the concrete suggestion.  I printed the bounds generated by `test_sample.TestRandint2.test_bound_float1`.
```
-3.2163061490620386 1.4340494179096543
-1.7548240925086693 4.447761323199037
-5.952811438090274 -2.784308792168858
0.7074908472677865 4.258527551050477
-4.63293410315047 1.7509332942724996
-3.907978778828104 4.91321954312264
-4.916231095357449 -2.146717804499249
-3.8030250737500784 5.786237847073696
-4.280589872674058 4.116831490893233
-3.259262529583008 4.1622474872584
```

Reply:
OK. Sorry I overlooked `test_bound_float1` is testing floating-point ranges.